docs/docs/ias.md

```diff
@@ -1,41 +1,49 @@
-calculated. It's a part of a serialized [`EnclaveInstanceInfo`](api/-conclave%20-core/com.r3.conclave.common/-enclave-instance-info/index.html) object and can be passed to clients from the server. This
-means clients don't need to contact Intel's servers themselves and thus don't need authorisation to use IAS.
+You can send and use the response anywhere, as Intel has signed it. When an enclave starts in debug or release mode, 
+it contacts the IAS to calculate the attestation. This attestation is a part of a serialized
+[`EnclaveInstanceInfo`](api/-conclave%20-core/com.r3.conclave.common/-enclave-instance-info/index.html) object and 
+can be passed to clients from the server. This means clients don't need authorization to use the IAS.
 
 ## Getting access
 
-IAS requires an authorised key to access it. This is distinct from the enclave signing key. To obtain an IAS access
-key: 
+You need an authorized key to access the IAS. This key is different from the enclave signing key.
+
+To obtain an IAS access key: 
 
-1. Sign the commercial use agreement [as part of getting a whitelisted enclave signing key](signing.md). You can run your application in simulation or debug mode without this, but try to do it well in advance of when you will need to run in release mode, as the application process can be lengthy.
-2. Get an Intel developer zone account and sign in [on the IAS API site](https://api.portal.trustedservices.intel.com/EPID-attestation).
-3. Subscribe to development and production access for "EPID Name Base Mode (Linkable Quotes)".
+1. Sign the commercial use agreement [as part of getting an enclave signing key](signing.md). You need to do this 
+   well in advance before using the release mode, as the application process can be lengthy. You can run 
+   your application in simulation or debug mode without this agreement.
+2. Get an Intel developer zone account and sign in to the [IAS API site](https://api.portal.trustedservices.intel.
+   com/EPID-attestation).
+3. Subscribe to development and production access for
+   [EPID Name Base Mode (Linkable Quotes)](https://api.portal.trustedservices.intel.com/EPID-attestation).
 
-!!! tip
-    You should give Intel a group email address rather than an individual address, as these will be used to send you
-    notification of security advisories affecting your system, so you know to upgrade and the timelines for 
-    [TCB recovery](renewability.md).
+!!!Note
+    
+    You should give Intel a group email address rather than an individual address, as Intel uses it to send security 
+    advisories that affect your system.
 
-Once you have access, you should be able to see an SPID and Primary Key associated with your subscription. You can supply these to the enclave when it is started:
+When you have access, you should be able to see an SPID and a primary key associated with your subscription. You can
```

Comment:
Maybe it would be nicer to clarify the acronym SPID  `to see a Service Provider ID (SPID)`

Reply:
Done.